File: 1293-three-consecutive-odds/three-consecutive-odds.py
class Solution:
    def threeConsecutiveOdds(self, arr: List[int]) -> bool:
        cnt = 0
        for i in arr:
            if i%2 != 0:
                cnt += 1
                if cnt == 3:
                    return True
            else:
                cnt = 0
        return cnt == 3

File: 1293-three-consecutive-odds/three-consecutive-odds.java
class Solution {
    public boolean threeConsecutiveOdds(int[] arr) {
        int n=arr.length;
        int k=0;
        for(int i=0;i<n;i++){
            if(arr[i]%2==1)
                k++;
            else 
                k=0;
            if(k>=3)
                return true;
            

        }
        return false;
    }
}